src/content/docs/de/tutorial/0-introduction/1.mdx

```diff
@@ -0,0 +1,60 @@
+---
+type: tutorial
+title: Über dieses Tutorial
+description: Wie du mit dem Tutorial "Baue deinen ersten Astro-Blog" anfangen kannst.
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+## Was muss ich wissen, um loszulegen?
+
+Wenn du ein paar Grundkenntnisse in **HTML**, **Markdown**, **CSS** und ein wenig **JavaScript** hast, dann kannst du direkt loslegen. 
+Du kannst das gesamte Tutorial abschließen, indem du den Anweisungen folgst. Astro ist für alle da! 🧑‍🚀 👩‍🚀 👨‍🚀 
+
+Außerdem brauchst du einen [GitHub](https://github.com)-Account (oder etwas Vergleichbares), um dein Projekt im Internet zu veröffentlichen. 
+
+<details>
+<summary>Wie kann ich die Checklisten am Ende jedes Kapitels nutzen?</summary>
+
+Du hakst sie einfach ab! 
+
+Am Ende jeder Seite befinden sich anklickbare Checklisten mit den Aufgaben, die du nun lösen können solltest. Hake diese Punkte ab, um deine Fortschritte im Tutorial-Tracker verfolgen zu können. 
+Der Tracker ist optional, kann dir aber dabei helfen, den Einstieg an der richtigen Stelle wieder zu finden, wenn du das Tutorial in mehreren Sitzungen abschließt. 
+Du kannst auch einige Punkte der Checkliste erst mal nicht abhaken lassen, als Erinnerung, falls du später zurückkehren möchtest. 
+(Diese Daten werden nur im lokalen Speicher deines Browsers gespeichert und nirgendwo sonst. Es werden keine Daten von Astro gesammelt.)
+
+</details>
+
+<details>
+<summary>Einheit 1 behandelt Themen, die ich schon kenne. Kann ich sie überspringen?</summary>
+
+Du kannst die Lektionen von [Einheit 1](/de/tutorial/1-setup/) nutzen, um sicherzugehen, dass du die Entwicklungstools und Accounts hast, die du brauchst, um das Tutorial abzuschließen. Die Einheit begleitet dich dabei, dein erstes Astro-Projekt zu erstellen, auf GitHub zu speichern und auf Netlify bereitzustellen. 
```

Comment:
```suggestion
Du kannst die Lektionen von [Einheit 1](/de/tutorial/1-setup/) nutzen, um sicherzugehen, dass du die Entwicklungstools und Accounts hast, die du brauchst, um das Tutorial abzuschließen. Die Einheit begleitet dich dabei, dein erstes Astro-Projekt zu erstellen, auf GitHub zu speichern und auf Netlify zu veröffentlichen. 
```

Reply:
Mich stört "bereitstellen" nicht, einfach weil gemäss Glossar "to deploy" mit "veröffentlichen" übersetzt werden soll.